entities/common.go

```diff
@@ -1,8 +1,24 @@
 package entities
 
+import (
+	"encoding/json"
+	"errors"
+)
+
 // Cursor represents the pagination cursor on many responses.
 type Cursor struct {
 	Limit  uint64 `json:"limit"`
-	After  uint64 `json:"after"`
-	Before uint64 `json:"before"`
+	After  string `json:"after"`
+	Before string `json:"before"`
+}
+
+// Epoch represents time.
+type Epoch int64
```

Comment:
You may not need this type.

Looking at the other types in the library, looks like util.UnixTimestamp already handles this.

Reply:
Please use `util.UnixTimestamp` for consistency.